src/main/java/frc/robot/subsystems/ClawSubsystem.java

```diff
@@ -6,26 +6,21 @@
 //Shuffleboard
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
-//pneumatic stuff
+//pneumatic libraries
 import edu.wpi.first.wpilibj.Compressor;
 import edu.wpi.first.wpilibj.PneumaticsModuleType;
 import edu.wpi.first.wpilibj.Solenoid;
 
 
-
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ClawSubsystem extends SubsystemBase {
-    
-    Compressor m_compressor = new Compressor(ClawConstants.kModuleID, PneumaticsModuleType.CTREPCM); //probably useless but whatever
+    //Defining the compressor might not be necessary for using it. It's also not giving out values when calling getPressure() on it
```

Comment:
Looked into this and we can remove this for future commits. The compressor object is already handled by the PCM by default and it's controlled by a analog pressure switch to the PCM. Only need to call it if we want to tell the PCM to run or shut off the compressor outside of the switch which is not recommended. See: https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/Compressor.html